exo-setup/src/docker-setup.ls

```diff
@@ -63,10 +64,11 @@ class DockerSetup
 
   # compiles list of names of dependencies a service relies on
   _get-service-dependencies: ->
-    dependencies = @_get-app-dependencies!
+    dependencies = []
```

Comment:
`_get-service-dependencies` and `_get-app-dependencies` should not have changed. There can be Exosphere applications which use another messaging bus like Nats, which would completely replace ExoCom. The concept of Exosphere needs to be separate from that of ExoCom, so that means we still need to read `application.yml` to see which messaging bus (a.k.a. application dependency) an app is using. 

Sorry I misinterpreted your question when you asked if every service uses ExoCom. When an Exosphere application uses ExoCom, then yes every service depends on ExoCom. But an application could also use another messaging bus in which case every service doesn't rely on ExoCom.